docs/how-to/enable-service-instances.rst

```diff
@@ -180,6 +181,37 @@ Finally, the client will be able to mount the NFS share:
 
    sudo mount -o rw -t nfs "nfs-bind-address:/fs-foo-dir /mnt
 
+Enable CephFS-mirror service
```

Comment:
Add inline formatting, i.e. Enable `cephfs-mirror` service

Reply:
idk if inline formatting should go here in a title